roles/dtc/common/templates/ndfc_route_control/ndfc_ip_access_list.j2

```diff
@@ -0,0 +1,357 @@
+! This NDFC Route_Control_ACLs config data structure is auto-generated
+! DO NOT EDIT MANUALLY
+!
+!
+{% if policy_types["ipv4_access_lists"] is defined %}
```

Comment:
As discussed please see if it makes sense to simplify this further by including additional child templates and split along technology lines (v4 and v6) for instance.